### DISCUSSION TITLE
Variable naming convention competition: `nameTypeLinkId`+`nameLinkId` VS `NameLinkId`+`nameLinkId`

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [deep-foundation/Discussions#32] [Polls]
How should we call variables in javascript when dealing with links? 
`nameTypeLinkId`+`nameLinkId` VS `NameLinkId`+`nameLinkId` 

Note: 
`nameTypeLinkId` and `NameLinkId` are used for types - links that are going to be used as types
`nameLinkId` are used for links - links that are not going to be used as types at least in the current scope

**Comment by suenot** [comment]
проголосовал
